src/main/steps/applicant1/hub-page/template.njk

```diff
@@ -32,6 +32,9 @@
 
         <h3 class="govuk-heading-s">{{ subHeading2 }}</h3>
         <p class="govuk-body">{{ line1 | safe }}</p>
+        {% if progressionIndex == 11 %}
+          <p class="govuk-body">{{ line2 | safe }}</p>
+        {% endif %}
```

Comment:
I asked Sanchita about these links and she said for now we can leave those out

Reply:
yh, i added the info to the 'Change description'